Quaver.Shared/QuaverGame.cs

```diff
@@ -210,6 +216,8 @@ protected override void Update(GameTime gameTime)
 
             QuaverScreenManager.Update(gameTime);
             Transitioner.Update(gameTime);
+
+            HandleSkinReloading();
```

Comment:
Why is this called in update? It should happen when the user presses the key combination.

Reply:
It's basically the same code as what is used for skin reloading in SettingsDialog.cs

The Load() function in SkinManager causes errors if you run it on a separate thread, and if you have it on the main thread directly after you FadeIn to black, it'll just block until the loading is done. So using the same code as what is in SettingsDialog, that skin reloading method handles smoothly Fading in and Out after the reloading of the skin.

Reply:
Gotcha. No worries then. However if it's the exact same functionality then we shouldn't be duplicating code and can just have one method for this used in both cases.